### PR TITLE
DATAGRAPH-1238 - Unify README.

### PR DESCRIPTION
**Pull request by michael-simons**
Please add additional fixes and improvements as needed and also decide, whether it can be passed down to 5.1 and 5.0 as well.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -11,81 +11,79 @@ The primary goal of the https://projects.spring.io/spring-data[Spring Data] proj
 
 The Spring Data Neo4j project aims to provide a familiar and consistent Spring-based programming model for integrating with the https://neo4j.com/[Neo4j] Graph Database.
 
-=== Features
+== Code of Conduct
```
Comment (mp911de) [inline README.adoc]: Let's move the CI badges to a dedicated file (see https://github.com/spring-projects/spring-data-mongodb/blob/master/CI.adoc). Same goes for the CI build instructions.

File: README.adoc [hunk 2]
```diff
@@ -104,141 +102,106 @@ public class MyConfiguration {
...
-Spring Data Neo4j will attempt to auto-configure itself using a file called `ogm.properties`, which it expects to find on root of the classpath. Here we will configure the HTTP Driver.
+Add the Maven dependency:
 
-[source,java]
+[source,xml]
 ----
-URI=http://user:password@localhost:7474
+<dependency>
+  <groupId>org.springframework.data</groupId>
+  <artifactId>spring-data-mongodb</artifactId>
```
Comment (mp911de) [inline README.adoc]: s/mongodb/neo4j

Comment (michael-simons) [reply README.adoc]: 🤦‍♂ Thanks.

**Comment by michael-simons** [comment]
Merge in Master with c9307c449b6bf15c6839e251f7f8aa34093d6e08, and in 5.1.x with 19001b0348f884cc5e0158fe640992f8c287113c

**Pull request closed without merging**